code/__HELPERS/bandetect.dm

```diff
@@ -0,0 +1,36 @@
+#define YOUNG 4
+
+
+/client/proc/join_date_check(var/y,var/m,var/d)
+	var/DBQuery/query = dbcon.NewQuery("SELECT DATEDIFF('[SQLdate()]','[y]-[m]-[d]')")
+
+	if(!query.Execute())
+		world.log << "SQL ERROR doing datediff. Error : \[[query.ErrorMsg()]\]\n"
+		return
+
+	if(query.NextRow())
+		var/diff = text2num(query.item[1])
+		world.log << diff
+		if(diff < YOUNG)
+			var/msg = "(IP: [address], ID: [computer_id]) is a new BYOND account made on [m]-[d]-[y]."
```

Comment:
Date should be in ISO-8601 format of YYYY-MM-DD so it aligns with the server time display